### PR TITLE
feat: move monitoring docs to team manual

### PR DESCRIPTION
**Pull request by madwort**

### CODE REVIEW

File: docs/timings-monitoring.md [hunk 1]
```diff
@@ -1,12 +1,3 @@
...
-    These notes are a work-in-progress.
-    This functionality is in limited release and subject to change.
-
-## Audience
-
-This documentation is aimed at researchers who want feedback on the performance of their job requests, as well as the performance context in which their job requests happen (i.e. was the server busy or quiet at the time).
-
 ## Timings
 
 Basic timing information is available from job and job request detail pages, for example:
```
Comment (lucyb) [inline docs/timings-monitoring.md:3]: Is it worth moving this into another part of the manual or removing the page altogether for the time-being? I wouldn't expect anyone to have linked to it and it's not currently providing the end user with much actionable information.

Comment (madwort) [reply docs/timings-monitoring.md:3]: yeah fair enough, will remove

**Review by lucyb — APPROVED** [review]

**Comment by StevenMaude** [comment]
If this is merged, it looks like #836 should be closed, without merging; is that correct?

**Comment by lucyb** [comment]
> If this is merged, it looks like #836 should be closed, without merging; is that correct?

That's correct. I'll merge this now, since Tom is away.

**Pull request merged**